lib/opensusebasetest.pm

```diff
@@ -637,6 +637,8 @@ sub wait_grub_to_boot_on_local_disk {
     }
 }
 
+sub grub_select;
```

Comment:
I would somehow expect this in the beginning of the file, after the `use` statements. If no one else see it like this. I don't have hard feelings.

Reply:
OK, this has been fixed. Can it now be merged?